R/hazardFunctions.R

```diff
@@ -20,23 +20,46 @@ ExpHazOS <- function(t, h01, h02, h12) {
   ((h12 - h02) * (h01 + h02) - h01 * h12 * exp(-h012 * t)) / ((h12 - h02) - h01 * exp(-h012 * t))
 }
 
+#' Helper Function for `avgHRExpOS()`
+#'
+#' @param x (`numeric`)\cr variable of integration.
+#' @param h01 (positive `number`)\cr transition hazard for 0 to 1 transition.
+#' @param h02 (positive `number`)\cr transition hazard for 0 to 2 transition.
+#' @param h12 (positive `number`)\cr transition hazard for 1 to 2 transition.
+#' @param h0 (`list`)\cr transition parameters for the first treatment group.
+#' @param h1 (`list`)\cr transition parameters for the second treatment group.
+#' @param alpha (`number`)\cr weight parameter, see `avgHRExpOS`.
+#' @return This returns the value of the integrand used to calculate
+#' the average hazard ratio for constant transition hazards, see `avgHRExpOS()`.
```

Comment:
```suggestion
#' @param alpha (`number`)\cr weight parameter, see [avgHRExpOS()].
#' @return This returns the value of the integrand used to calculate
#' the average hazard ratio for constant transition hazards, see [avgHRExpOS()].
```
advantage of this is that you get a link in the help page